cmd/subcmds/output.go

```diff
@@ -94,25 +80,30 @@ func pickWriter(args *inArgs, data *bytes.Buffer) (ir.Writer, error) {
 		return csvio.NewWriter(w), nil
 	case mdOutputFormat:
 		return mdio.NewWriter(w), nil
-	case apiOutputFormat:
-		return confio.NewWriter(w, args.configFile)
-	default:
-		return nil, fmt.Errorf("bad output format: %q", args.outputFmt)
+	case jsonOutputFormat:
+		if isSynth {
```

Comment:
don't we want a json writer for optimize? It can really help you validate optimization results with the analyzer

Reply:
#210 
should change the output algorithm (it is different than other fmt writers)